client/utils/rest.go

```diff
@@ -171,14 +184,23 @@ func (br BaseReq) ValidateBasic(w http.ResponseWriter) bool {
 		}
 	}
 
-	if len(br.Name) == 0 {
+	if len(br.From) == 0 {
 		WriteErrorResponse(w, http.StatusUnauthorized, "name required but not specified")
 		return false
 	}
 
 	return true
 }
 
+// ToTxBuilder returns a TxBuilder based off of the populated fields of a BaseReq.
+// Any additional required fields must be provided as parameters.
+func (br BaseReq) ToTxBuilder(cdc *codec.Codec, gas uint64, gasAdj float64) authtxb.TxBuilder {
```

Comment:
I'm thinking maybe this doesn't have to be a method on `BaseReq`, but rather could also be just an auxiliary function that takes a `BaseReq` as param.

Reply:
I like defining this on BR Good addition.